index.bs

```diff
@@ -2898,3 +2902,18 @@ To decode a PSK `P` given a QR code, follow these steps:
 
 1. Obtain the string `N` by decoding the QR code.
 2. Parse `N` as a hexadecimal number to obtain `P`.
+
+Appendix D: Media Time Conversions {#appendix-d}
+==================================
+
+To convert between a media synchronization timestamp for a given audio or video
+frame and a media timeline value, the following formula can be used:
+
+```
+media-timeline-value = media-zero-time + (value / scale)
+```
+
+Where:
+- `media-zero-time` is the origin of the [=media timeline=] as defined in HTML.
+- `value` and `scale` are the values passed in the `sync-time` field of the
```

Comment:
Might be pendantic, but might be worth suggesting how the math should be calculated since value and scale are both `uint` types and media-timeline-value is presumably a `float64`. 

My guess is that both value and scale should be converted to `float64` before division occurs in order to avoid rounding?

Reply:
Added specific details about how `media-timeline-value` and the computation should be done, and also what to do in event of an overflow (pin to the max value).